go/desktop/web.config

```diff
@@ -7,7 +7,7 @@
 
                 <!-- Links for Desktop 10.0 onward -->
```

Comment:
To help prevent us from updating the wrong web.config file later, can you add a note that 14.0 onward updates should go in `windows/web.config`?